README.md

```diff
@@ -1,215 +1,126 @@
-# Spring Web MVC Lab 1
-
-Congratulations! You have been hired by Access Camp and for your first task, you
-have been tasked with building out a website to log campers with their
-activities.
-
-Your task is to build the API according to the deliverables below.
-
-## Setup
-
-Use [Spring Initializr](https://start.spring.io/) to create a project with the
-following properties:
-
-- Java 11
-- Maven Build
-
-And add the following dependencies:
-
-- Spring Web.
-- Spring Data JPA.
-- A database of your choice (H2, Postgres, MySQL).
-
-Generate the project and open it up in IntelliJ.
-
-## Models
-
-You need to create the following relationships:
-
-- A `Camper` has many `Signups`, and has many `Activity`s through `Signup`s
-- An `Activity` has many `Signups`, and has many has many `Camper`s through
-  `Signup`s
-- A `Signup` belongs to a `Camper` and belongs to a `Activity`
-
-Start by creating the models and migrations for the following database tables:
-
-![domain diagram](https://curriculum-content.s3.amazonaws.com/phase-4/mock-challenge-camping-fun/diagram.png)
-
-## Validations
-
-Add validations to the `Camper` model:
-
-- must have a `name`
-- must have an `age` between 8 and 18
-
-Add validations to the `Signup` model:
-
-- must have a `time` between 0 and 23 (referring to the hour of day for the
-  activity)
-
-## Routes
-
-Set up the following routes. Make sure to return JSON data in the format
-specified along with the appropriate HTTP verb.
-
-### GET /campers
-
-Return JSON data in the format below. **Note**: you should return a JSON
-response in this format, without any additional nested data related to each
-camper.
-
-```json
-[
-  {
-    "id": 1,
-    "name": "Caitlin",
-    "age": 8
-  },
-  {
-    "id": 2,
-    "name": "Lizzie",
-    "age": 9
-  }
-]
+# Spring Web MVC Lab
+
+## Learning Goals
+
+- Practice creating a web application using the Spring Framework.
+- Create a `@Controller` class.
+- Create a `@Service` class.
+- Use Postman as an API Client tool to test.
+
+## Instructions
+
+Create a simple web application using the Spring Framework to perform simple
+calculator operations between two numbers.
```

Comment:
Not clear what "numbers" implies.  The examples show integers, but return type is double.